css/about/footer.css

```diff
@@ -0,0 +1,19 @@
+footer {
+  padding: 2rem;
+  display: flex;
+  gap: 1rem;
+  justify-content: space-around;
+  align-items: center;
+  background-color: var(--primary-color);
+}
+
+footer img {
+  width: 200px;
+  background-color: #ccc;
+  border-radius: 5px;
+}
+
+footer p {
```

Comment:
- **About: Mobile size**:
- It has a white background.

![Capture1](https://user-images.githubusercontent.com/63972670/203524091-0ced8cd0-6a10-4e9d-b007-e4ec86b773be.PNG)
- See Your Design:

![Capture1](https://user-images.githubusercontent.com/63972670/203524452-06cd6a1f-b9e8-4234-9f49-15c34248ff6a.PNG)